### ISSUE TITLE
[💡 Feature Request]: Logging all activity

### ISSUE THREAD

**Issue by mschindler504** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#627]
### What's the Problem?

Would love to have a log of all the commands, responses, and times of each. Would be great for referencing, troubleshooting, and a source of information that could be used/parsed and incorporated in other personal projects (get the response and send result to an API/device/email/etc.).

### What's your solution or idea?

Save all command strings and append to a log file with the time. Same with the result... save and append to a log file with the time. Should be able to do this before or after the result is displayed in the innerHTML. 

### Alternatives Considered

Store data in json file for even easier, specified data retrieval. Or possibly in a sql lite database file. These options would actually be better than a log file, just a little more work. But if you would like help, I'd be willing to work with you. Love what you have done so far!

### App Version

Built from source

### OS Version

Windows 10

### Additional context

An API or command line option to send and receive commands/results would be great as well. 
Basically I have a home website that controls all my RF devices, and this application now gives me the capability and a doorway to control all my 'smart'er devices that are connected to my Google Home/Assistant, from my website. I have an automation script that types the appropriate command to toggle a light/etc. after a button is clicked. Just looking for a way to capture and send the results back to my web page.

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%E2%9C%A8+enhancement%22+) and didn't find any duplicate issue
- [X] This issue only covers one particular feature request

**Comment by mschindler504** [comment]
This is not a bug, just a feature request
